examples/_debug/legends/size-continuous-style.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>GeoJSON Layer with CARTO Tiles Example</title>
+
+    <!-- Custom CSS -->
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.css"
+    />
+    <link
+     rel="stylesheet"
+     type="text/css"
+     href="https://libs.cartocdn.com/airship-style/v2.4/airship.min.css"
+   />
+
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+      }
+
+      #map {
+        width: 100vw;
+        height: 100vh;
+      }
+    </style>
+  </head>
+
+  <body>
+    <section id="map"></section>
+    <div class="as-map-panels">
+      <div class="as-panel as-panel--top as-panel--left">
+        <as-legend>
+          <as-legend-size-continuous id="legend" slot="legends"></as-legend-size-continuous>
+        </as-legend>
+      </div>
+    </div>
+
+    <!-- Map libraries-->
+    <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.js"></script>
+    <script src="https://libs.cartocdn.com/airship-components/v2.4/airship.js"></script>
+    <script src="https://unpkg.com/deck.gl@8.2.0/dist.min.js"></script>
+
+    <script src="/dist/umd/index.min.js"></script>
+
+    <script>
+      async function initialize() {
+        carto.auth.setDefaultCredentials({ username: 'cartoframes' });
+
+        const deckMap = carto.viz.createMap({
+          basemap: 'voyager',
+          view: {
+            longitude: -116.846099,
+            latitude: 50.2672297,
+            zoom: 2.5
+          }
+        });
+
+        const style = carto.viz.style.sizeContinuous('total_pop', {
+          rangeMin: 100000,
+          rangeMax: 5000000,
+          sizeRange: [5, 50],
+          color: '#0000FF',
+          strokeColor: '#40E0D0',
+          strokeWidth: 1
+        });
+
+        const tempLayer = new carto.viz.Layer('ca_measures', style);
+        tempLayer.addTo(deckMap);
+        new carto.viz.Legend('#legend', tempLayer);
+      }
+
+      function initializeLines() {
+        carto.auth.setDefaultCredentials({ username: 'cartoframes' });
+
+        const deckMap = carto.viz.createMap({
+          basemap: 'voyager',
+          view: {
+            longitude: 135.7,
+            latitude: -25.48,
+            zoom: 3
+          }
+        });
+
+        const style = carto.viz.style.sizeContinuous('length_km', {
+          // bins: 3,
+          // sizeRange: [1, 5]
+        });
+        const layer = new carto.viz.Layer('roads', style);
+        layer.addTo(deckMap);
+        new carto.viz.Legend('#legend', layer);
+      }
+
+      initialize();
+      // initializeLines();
```

Comment:
I've noticed that this legend, for lines, get too wide:

![image](https://user-images.githubusercontent.com/458196/88904476-695d5980-d255-11ea-833f-43cf14db8654.png)

Not an issue here, but on the Airship side...